### DISCUSSION TITLE
Re-add "Close" button next to "Refresh" in extra networks, and possibly turn them into emojis

### DISCUSSION THREAD

**Discussion by hollowstrawberry** [vladmandic/sdnext#307] [Ideas]
Before automatic1111 slowed down, I saw a post there asking for them to replace the "Refresh" and "Close" buttons in the extra networks menu with emojis 🔃 ❌ for stylistic reasons. Instead the "Close" button was later removed, making it so that you have to press 🎴 up above to close the menu. This is slightly cumbersome. Hopefully you agree that this small change would be worth it.

Refer to the image to see what I mean.

![image](https://user-images.githubusercontent.com/33796679/233451528-ba097ab7-c722-45af-a563-1a99374faec8.png)

**Comment by vladmandic** [comment]
i'd be ok with that, but no time to search what exactly was removed in the code. if you can point that out or (even better) contribute with a pr, that would work.

**Reply by hollowstrawberry** [reply]
Of course, I forget that's how open source works. Here you go https://github.com/vladmandic/automatic/pull/312
